CanonicalLtd/juju-cards
# Updated the infrastructure of the project

## Done
Replaced gulp and old deps with npm scripts and updated deps.

## QA
- Run `npm install`
- Run `npm run build`
- Open the file at `build/index.html`
- Check that cards load and are styled
